runtime-sdk/src/modules/core/mod.rs

```diff
@@ -299,6 +367,9 @@ impl module::MethodHandler for Module {
                 let args = cbor::from_value(args).map_err(|_| Error::InvalidArgument)?;
                 Ok(cbor::to_value(&Self::query_estimate_gas(ctx, args)?))
             })()),
+            BATCH_WEIGHT_LIMIT_QUERY_METHOD => {
+                module::DispatchResult::Handled(Ok(cbor::to_value(&Self::get_weight_limits(ctx))))
+            }
```

Comment:
Should we handle this in the dispatcher and make it so that any module can provide per-batch weight limits? E.g., we could extend `BlockHandler` with `get_block_weight_limits` and invoke this in the `Dispatcher` instead of the core module?

Reply:
Yeah makes sense, done